aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -935,6 +935,14 @@ Tensor istft(const Tensor& self, const int64_t n_fft, const optional<int64_t> ho
   if (input_dim == 3) {
     y = y.squeeze(0);
   }
+  // zero padding if the given lengthOpt is longer than expected
+  if(end > expected_output_signal_len) {
+    TORCH_WARN_ONCE(
+      "The length of signal is shorter than the length parameter. Pad zeros to the tail. "
+      "Please check your center and hop_length settings."
```

Comment:
This sounds more like a suggestion for the user to pad with zeros, how about:
```suggestion
      "The length of signal is shorter than the length parameter. Result is being padded with zeros in the tail. "
      "Please check your center and hop_length settings."
```

Reply:
Same in a776165